samples/sample-generic-content-sidekick-plugin/src/components/sample-generic-content-sidekick-plugin-item/component.tsx

```diff
@@ -16,9 +17,14 @@ function SampleGenericContentSidekickPlugin(
   BbbPluginSdk.initialize(uuid);
   const pluginApi: PluginApi = BbbPluginSdk.getPluginApi(uuid);
 
+  const GENERIC_CONTENT_BADGE_ID = 'first-sidekick-component';
+
+  const [countGenericContent, setCountGenericContent] = React.useState(1);
```

Comment:
This could be a reference instead of a state and starting at 0:
```suggestion
  const countGenericContent = React.useRef(0);
```